upstairs/src/lib.rs

```diff
@@ -603,33 +626,33 @@ async fn cmd_loop(
                  * check.
                  */
                 let more = io_send(up, &mut fw, up_coms.client_id, lossy).await?;
-                if more {
-                    resume = deadline_secs(1);
-                    if !flow_control {
-                        println!("[{}] flow control start ", up_coms.client_id);
-                        flow_control = true;
-                    }
+                if more && !flow_control {
+                    println!("[{}] flow control start ", up_coms.client_id);
+                    flow_control = true;
                 }
             }
-            _ = sleep_until(resume), if flow_control => {
+            _ = sleep_until(deadline_secs(5)), if flow_control => {
```

Comment:
The old time of 1 second was probably too long if we are in a flow control situation.
5 seconds I'm sure is too long to wait.  I think we should put it back to 1 second but
(XXX) more research will be required to figure out how long to wait.

Reply:
Done in 273a0a2